functions/airtable-shared-view.js

```diff
@@ -95,4 +99,11 @@ function parseAirtableData(airtableData) {
   return data;
 }
 
-module.exports = { readAirtableSharedView, parseAirtableData };
+const unitTestFunctions = {
```

Comment:
I might just export the functions you wanna test rather than doing so under a "unitTestFunctions" and then test the translators separately.

Reply:
Sounds good, I don't even remember why I tried to make a separate object for this lol